src/renderer/runner.ts

```diff
@@ -350,32 +373,23 @@ export class Runner {
   }
 
   /**
-   * Execute Electron.
-   *
-   * @param {string} dir
-   * @returns {Promise<RunResult>}
-   * @memberof Runner
+   * Actually runs the fiddle with either local electron build
+   * or the user selected electron version
    */
-  public async execute(dir: string): Promise<RunResult> {
-    const {
-      currentElectronVersion,
-      flushOutput,
-      pushOutput,
-      executionFlags,
-    } = this.appState;
-
-    const { version, localPath } = currentElectronVersion;
-    const binaryPath = getElectronBinaryPath(version, localPath);
-    console.log(`Runner: Binary ${binaryPath} ready, launching`);
-
+  private async runFiddle(params: RunFiddleParams): Promise<RunResult> {
+    const { localPath, isValidBuild, version, dir } = params;
+    const { runner, pushOutput, flushOutput } = this.appState;
+    const fiddleRunner = await runner;
     const env = this.buildChildEnvVars();
 
-    // Add user-specified cli flags if any have been set.
-    const options = [dir, '--inspect'].concat(executionFlags);
```

Comment:
Any reason why this was removed, or was this re-added in a different area? We do want to have the ability to inject cli flags here if we can

Reply:
Yup, nice catch! Missed that!